sru/csrc/sru_cuda_impl.cpp

```diff
@@ -276,8 +494,13 @@ void sru_bi_backward(
 }
 
 PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
+  m.def("sru_forward_simple", &sru_forward_simple, "SRU forward (CUDA version)");
+  m.def("sru_backward_simple", &sru_backward_simple, "SRU backward (CUDA version)");
+  m.def("sru_bi_backward_simple", &sru_bi_backward_simple, "SRU bidirectional backward (CUDA version)");
+  m.def("sru_bi_forward_simple", &sru_bi_forward_simple, "SRU bidirectional forward (CUDA version)");
```

Comment:
can we put bi_forward above bi_backward, for consistency with the ordering of single forward and single backward?